server/views/pages/courtAppearance/save-court-case.njk

```diff
@@ -0,0 +1,33 @@
+{% extends "../../partials/prisonerBannerBackLayout.njk" %}
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - Successfully saved" %}
+
+{% block content %}
+    {{ super() }}
+    <div class="govuk-grid-row govuk-!-margin-top-4">
+        <div class="govuk-grid-column-two-thirds">
+            <span class="govuk-caption-l">
+                Add a court case
+            </span>
+            <h2 class="govuk-heading-l">Court case on {{ courtAppearance.warrantDate | formatDate }} at {{ courtAppearanceCourtName }} has been saved</h2>
+            {{ govukInsetText({
+                text: "This appearance has not been saved to NOMIS"
+            }) }}
+            <p class="govuk-body">
+                Court cases can only be saved to NOMIS when <a href="/person/{{ nomsId }}/{{ addOrEditCourtCase }}/{{ courtCaseReference }}/{{ addOrEditCourtAppearance }}/{{ appearanceReference }}/task-list">all sections on the task list</a> are complete.
+            </p>
+            <p class="govuk-body">
+                You can return to finish adding court case and offence information from <a href="/person/{{ nomsId }}/">{{ prisoner | firstNameSpaceLastName | pluraliseName }} Court cases page</a>.
+            </p>
+            <form method="post" action="/person/{{ nomsId }}/">
```

Comment:
there is no post route on this, will this work? I think you can define the govukButton with a href value and it will turn into an a element

Reply:
I've removed this and used href instead